src/components/item/item.ts

```diff
@@ -354,4 +362,24 @@ export class Item {
       icon.addClass('item-icon');
     });
   }
+
+  /**
+   * @private
+   */  
+  height(): number {
+    return this._elementRef.nativeElement.offsetHeight;
```

Comment:
@adamdbradley here: https://github.com/driftyco/ionic/pull/6937/files#diff-4ad8cd6371a10686d79bd259ccef2897R34

it is 100% unavoidable, i don't think it is a problem... it is only called on touchstart